### DISCUSSION TITLE
Ollama with API key

### DISCUSSION THREAD

**Discussion by ryoheiszk** [langgenius/dify#7403] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Hello, I'm seeking advice on using a specific AI model setup with Dify. To clarify my situation:

I am a user of the system, not a developer or system administrator.
There is a server running phi3 model via Ollama in a global (internet-accessible) environment.
The server requires authentication with an API key for access.

Current situation:

I can successfully use this setup in VSCode with the Continue extension by specifying the API key.
However, in Dify, I don't see a clear option to input this API key.

My question:
How can I properly connect to and use this Ollama-run phi3 model in Dify? Specifically, where and how should I input the API key or other necessary authentication details?
Any step-by-step guidance or tips would be greatly appreciated, especially considering I'm not a technical administrator of the system.
Thank you in advance for your help!

![2024-08-19_18h53_58](https://github.com/user-attachments/assets/2e2626ff-26ed-40aa-a1c0-9e5b6c0f1a94)

**Comment by SeekPoint** [comment]
+1
